homeassistant/components/switch/pilight.py

```diff
@@ -0,0 +1,216 @@
+"""
+
+Support for Pilight Switches.
+"""
```

Comment:
Can you please include details for the configuration (especially a sample for the configuration, yaml file like [this](https://home-assistant.io/components/switch.command_switch/)) till this platform has it's own documentation.